mod_auth_cas fails to build against CentOS/RHEL 7 Apache

When building against the latest git clone of mod_auth_cas the build fails as can be seen below.  If I git clone -b 'v1.2' the version builds just fine.

make[2]: Entering directory `/tmp/mod_auth_cas/src'
/usr/bin/apxs -c -lpcre -lcrypto  -Wc," -D_LARGEFILE64_SOURCE -g -O2 -Wall -Wextra -Wdeclaration-after-statement -Wformat -Wformat-security -Wmissing-declarations -Wno-unused-parameter -Wpointer-arith -Wstrict-prototypes " -Wl,"  -lcurl   -lpcre -lcrypto " mod_auth_cas.c cas_saml_attr.c
/usr/lib64/apr-1/build/libtool --silent --mode=compile gcc -std=gnu99 -prefer-pic -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic  -DLINUX -D_REENTRANT -D_GNU_SOURCE -pthread -I/usr/include/httpd  -I/usr/include/apr-1   -I/usr/include/apr-1   -D_LARGEFILE64_SOURCE -g -O2 -Wall -Wextra -Wdeclaration-after-statement -Wformat -Wformat-security -Wmissing-declarations -Wno-unused-parameter -Wpointer-arith -Wstrict-prototypes   -c -o mod_auth_cas.lo mod_auth_cas.c && touch mod_auth_cas.slo
mod_auth_cas.c:49:24: fatal error: apr_escape.h: No such file or directory
 #include "apr_escape.h"
                        ^
compilation terminated.
apxs:Error: Command failed with rc=65536
.
make[2]: Leaving directory `/tmp/mod_auth_cas/src'
make[2]: *** [mod_auth_cas.la] Error 1
make[1]: Leaving directory `/tmp/mod_auth_cas'
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
The command '/bin/sh -c cd /tmp && git clone https://github.com/apereo/mod_auth_cas.git &&     cd /tmp/mod_auth_cas &&     autoreconf -ivf  &&     ./configure &&     make && make install' returned a non-zero code: 2

--- a/README
+++ b/README
@@ -45,8 +45,8 @@ Building from Source
 
 The following development libraries and utilities must be installed:
 * OpenSSL - 0.9.8c or higher
-* Apache Portable Runtime - 1.2.8 or higher
-* Apache Portable Runtime Utilities - 1.2.7 or higher
+* Apache Portable Runtime - 1.5.0 or higher
+* Apache Portable Runtime Utilities - 1.3.0 or higher
 * Apache Web Server - 2.2.3 or higher
 * libcurl - 7.18.2 or higher
 * libpcre - 7.8 or higher

